### DISCUSSION TITLE
Guess the song 🎸

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1025] [Galaxy Trivia] [answered]
```
e|------------------------------------|
B|------------------------------------|
G|-2-2--2--2--2--2---------5-----5b---|
D|-2-2--2--2--2--2----5----2-----2----|
A|------------------0---0-----0-------|
E|------------------------------------|
```

**Bonus:** Name the songwriter.

**Comment by peterbaccaro** [comment, marked as answer]
The song is **La Grange** by the American rock band **ZZ Top**.  It was released as the first single from their 1973 album Tres Hombres and has since become one of the band's signature songs.

The song was written by the bands guitarist and vocalist **Billy Gibbons** and it showcases their blues-rock sound with a touch of Southern rock influence.

**Comment by Ben74x** [comment]
Ironman by Black Sabbath!

**Reply by ParanoidUser** [reply]
Nice try! You're on the right track. 😉
